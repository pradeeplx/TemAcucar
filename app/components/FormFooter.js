import React from 'react'
import { View } from 'react-native'

export default FormFooter = ({ children }) => (
  <View style={{
    alignSelf: 'stretch',
    alignItems: 'center',
    marginTop: 24,
  }}>
    {children}
  </View>
)
