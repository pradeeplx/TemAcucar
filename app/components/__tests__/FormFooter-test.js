import 'react-native'
import React from 'react'
import FormFooter from '../FormFooter'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <FormFooter>Footer</FormFooter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
