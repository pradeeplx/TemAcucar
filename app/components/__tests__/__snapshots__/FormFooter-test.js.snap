exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "marginTop": 24,
    }
  }>
  Footer
</View>
`;
